code/game/machinery/computer/_computer.dm

```diff
@@ -19,6 +19,8 @@
 	var/time_to_unscrew = 2 SECONDS
 	/// Are we authenticated to use this? Used by things like comms console, security and medical data, and apc controller.
 	var/authenticated = FALSE
+	/// Will projectiles be able to pass over this computer?
+	var/projectiles_pass_chance = 75
```

Comment:
Maybe lower to 50%? 75 feels awfully high for something that's supposed to act as cover

Reply:
50% is barricades like sandbags, methinks they should be more effective at that since they're dedicated to acting as cover

Reply:
will be 65% after discussing in discord (here for history reasons)